imports/server/api/resources/users.ts

```diff
@@ -37,13 +37,14 @@ const ACTIVE_THRESHOLD = 365 * 24 * 60 * 60 * 1000;
 const renderUser = function renderUser(user: Meteor.User, profile: ProfileType) {
   const active = user.lastLogin &&
           Date.now() - user.lastLogin.getTime() < ACTIVE_THRESHOLD;
+  const operator = deprecatedUserMayMakeOperator(user._id);
```

Comment:
@grantaelliott as the maintainer of the only known jolly-roger API client: do you know if you actually rely on the presence/value of the `operator` field from the jolly-roger API for the POTW site or any of the other hunt sites you've written?  Would love to have the flexibility to drop this field or restructure it on a per-hunt basis so long as it wouldn't break you.

Reply:
This is why we should switch to graphql :joy:

Reply:
POTW does not depend on the operator field.